### DISCUSSION TITLE
Error (400) in the Dataset's Dropbox

### DISCUSSION THREAD

**Discussion by SSHurtado** [ARCLab-MIT/STORM-AI-devkit-2025#4] [Bugs Report] [answered]
![image](https://github.com/user-attachments/assets/d78108ba-329d-4e17-aa8f-43661ce6af3e)

@ArmaanSoniQG I tried viewing the dataset and it says Error (400) in the Dataset's Dropbox

**Comment by hlizsolera** [comment, marked as answer]
This issue was caused by an HTML formatting error and has since been resolved.
